locations/spiders/vakifbank_tr.py

```diff
@@ -0,0 +1,46 @@
+import json
+
+import scrapy
+from scrapy.http import JsonRequest
+
+from locations.categories import Categories, apply_category
+from locations.items import Feature
+
+
+class VakifBankTRSpider(scrapy.Spider):
+    name = "vakifbank_tr"
+    item_attributes = {"brand": "Vakıfbank", "brand_wikidata": "Q1148521"}
+    allowed_domains = ["vakifbank.com.tr"]
+    requires_proxy = "TR"
+    no_refs = True
+
+    def start_requests(self):
+        url = "https://maps.vakifbank.com.tr/API/api/v1/Search/GetAllBranchAndAtmWithFiltersObj"
+        payload = {
+            "CountryId": 0,
+            "WhichOne": 1,  # Branches
+            "BranchNameOrCode": "",
+            "CityId": -1,
+            "TownId": -1,
+            "NeighborhoodId": -1,
+            "Options": [],
+        }
+        yield JsonRequest(url=url, method="POST", body=json.dumps(payload), callback=self.parse)
```

Comment:
If you set `data` to `payload` in the `JsonRequest` ([doc](https://docs.scrapy.org/en/latest/topics/request-response.html#scrapy.http.JsonRequest)), it will do the JSON serialization for you.

Reply:
Thanks!